docs/quark/README.md

```diff
@@ -1,54 +0,0 @@
-# Quark
-
-Quarks are the fundamental building blocks of atoms through which DNA, life,
-and matter arise. Similarly this package is the core framework for constructing
-the atom tokens which will power [The Cosmos Network](https://cosmos.network/).
-
-The Quark framework affords you all the tools you need to rapidly develop
-robust blockchains and blockchain applications which are interoperable with The
-Cosmos Hub. Quark is an abstraction of [Tendermint](https://tendermint.com/)
-which provides the core consensus engine for your blockchain. Beyond consensus,
-Quark provides a blockchain development 'starter-pack' of common blockchain
-modules while not enforcing their use thus giving maximum flexibility for
-application customization. For example, do you require fees, how do you
-want to log messages, do you enable IBC, do you even have a cryptocurrency?
-
-Disclaimer: when power and flexibility meet, the result is also some level of
-complexity and a learning curve.  Here is an introduction to the core concepts
-embedded in Quark.
-
-## Inspiration
```

Comment:
Do we really want to remove the inspiration section? I kind of like it! Maybe it could be formalized a bit... your call

Reply:
didn't know where to put it. how about at the end of the main README?

Reply:
i put it there, with slight modifications